aim/api/resource.py

```diff
@@ -291,6 +291,225 @@ def __init__(self, **kwargs):
                                                     **kwargs)
 
 
+class SpanVsourceGroup(AciResourceBase):
+    """Resource representing a ERSPAN VSource Group in ACI.
+
+    Identity attributes are RNs for ERSPAN VSource Group.
+    """
+
+    identity_attributes = t.identity(
+        ('name', t.name))
+    other_attributes = t.other(
+        ('admin_st', t.enum("start", "stop")),
```

Comment:
What does st stands for here, status? Suggest to spell out the full name to make it clear.

Reply:
okay will change that. It is "admin state"